docs/rest-api.md

```diff
@@ -161,7 +161,7 @@ freqtrade-client --config rest_config.json <command> [optional parameters]
 | `delete_lock <lock_id>` | Deletes (disables) the lock by id.
 | `locks add <pair>, <until>, [side], [reason]` | Locks a pair until "until". (Until will be rounded up to the nearest timeframe).
 | `profit` | Display a summary of your profit/loss from close trades and some stats about your performance.
-| `forceexit <trade_id>` | Instantly exits the given trade  (Ignoring `minimum_roi`).
+| `forceexit <trade_id> [order_type] [rate]` | Instantly exits the given trade (ignoring `minimum_roi`), using the given order type ("market" or "limit", uses your config setting if not specified), and the chosen rate (full sell if not specified).
```

Comment:
on the risk of sounding picky ... I'm tempted to say that the 3rd parameter is amount, not rate... 😆 

https://github.com/freqtrade/freqtrade/blob/7f210ab3b54e1e78d77297fde8a676ebc56f5bab/ft_client/freqtrade_client/ft_rest_client.py#L331

Reply:
Ah, my mistake!